ui/tutorial-renderer/src/main/res/layout/tutorial_tips_start.xml

```diff
@@ -5,69 +5,19 @@
         <variable name="callbacks" type="org.cru.godtools.tutorial.TutorialCallbacks" />
     </data>
```

Comment:
we no longer use `callbacks` within the data binding, so you can remove that and the empty `<data>` xml node as well.